app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -3679,7 +3683,6 @@ class StateFragmentTest {
       // values being different.
       verifyViewTypeIsPresent(MATH_EQUATION_INPUT_INTERACTION)
       verifyContentContains("any equivalent expression")
-      verifySubmitAnswerButtonIsDisabled()
```

Comment:
Here and everywhere else, why do we need to remove this?

Reply:
Because now submit button is enabled after wrong answer submitted by the user.

Reply:
Could you update the tests to verify that the button is enabled, instead? I suggest swapping all the previous removed lines with:

```kotlin
verifySubmitAnswerButtonIsEnabled() // Wrong answers shouldn't disable submit.
```

Reply:
I will add, thank you.